### ISSUE TITLE
test_caffe_model

### ISSUE THREAD

**Issue by 3DMM-ICME2023** [ramprs/grad-cam#13]
Thanks a lot for sharing the very nice work! @ramprs  @abhshkdz 
How can i get the Gradient class activation maps with the model trained with caffe rather than pytorch models?
Moreover, could you please tell me how to generate the wonderful results as this website shows(https://medium.com/twentybn/visual-explanation-for-video-recognition-87e9ba2a675b)
Thank you very much!

**Comment by abhshkdz** [comment]
> How can i get the Gradient class activation maps with the model trained with caffe rather than pytorch models?

The classification code in this repository is for generating Grad-CAM using Caffe-pretrained models (using `loadcaffe`), that might be useful. For pytorch-compatibility, check out [1], [2], [3].

> Moreover, could you please tell me how to generate the wonderful results as this website shows(https://medium.com/twentybn/visual-explanation-for-video-recognition-87e9ba2a675b)

By generating Grad-CAM on individual frames from the video clip, and stitching all the image-overlaid maps together back into a video.

[1]: https://github.com/jacobgil/pytorch-grad-cam
[2]: https://github.com/kazuto1011/grad-cam-pytorch
[3]: https://github.com/meliketoy/gradcam.pytorch
